Src/EmbeddedScripts.JS.ClearScriptV8/V8ScriptEngineExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Linq;
+using EmbeddedScripts.Shared;
+using Microsoft.ClearScript.V8;
+
+namespace EmbeddedScripts.JS.ClearScriptV8
+{
+    public static class V8ScriptEngineExtensions
+    {
+        internal static V8ScriptEngine AddHostObjectByAlias(this V8ScriptEngine engine, Container container,
+            string alias)
+        {
+            var type = container.GetTypeByAlias(alias);
+            var instance = container.Resolve(alias);
+
+            if (type.IsPrimitive || type == typeof(string))
+                engine.Script[alias] = instance;
+            else
+                engine.AddHostObject(alias, instance);
```

Comment:
Will AddHostObject for primitive value throw an error?

Reply:
No, I have tests for this case.

Reply:
Yes. This method doesn't work with primitives and strings (https://github.com/microsoft/ClearScript/issues/90#issuecomment-447082988)